spec/factories/host.rb

```diff
@@ -2,7 +2,7 @@
   factory :host do
     sequence(:name)     { |n| "host_#{seq_padded_for_sorting(n)}" }
     sequence(:hostname) { |n| "host_#{seq_padded_for_sorting(n)}" }
-    vmm_vendor          "vmware"
+    vmm_vendor_display  "VMware"
```

Comment:
I think the setter for vmm_vendor_display is weird.  Ideally, the code should work with the raw column, and vmm_vendor_display should only be used for display / UI purposes.  Having "display" knowledge spread around (like in factories and in the refresh parser, feels wrong).